test/integration/auth.spec.ts

```diff
@@ -36,6 +36,7 @@ import {
 } from '../../lib/auth/index';
 import * as sinon from 'sinon';
 import * as sinonChai from 'sinon-chai';
+import { ActionCodeSettings } from '../../src/auth/action-code-settings-builder';
```

Comment:
Are we able to import this type from `../../lib/` since these are integration tests?

Reply:
Updated.